### PR TITLE
docs(release-checklist.md): update with latest processes

### PR DESCRIPTION
**Pull request by vdice**
Amending as we release beta4...

Closes https://github.com/deis/jenkins-jobs/issues/79

### CODE REVIEW

File: src/roadmap/release-checklist.md [hunk 1]
```diff
@@ -34,43 +34,59 @@ A release consists of the following artifacts:
...
-Next, we'll create a new [Helm Classic](https://github.com/helm/helm-classic) chart so that we can "stage" a
-version of our release for testing. The process is fairly simple:
-
-1. Create a new branch: `git checkout -b release-$WORKFLOW_RELEASE`
-2. Copy the existing `dev` chart: `cp -r workflow-dev workflow-$WORKFLOW_RELEASE_SHORT`
-3. Modify the `workflow-$WORKFLOW_RELEASE_SHORT/tpl/generate_params.toml` file to ensure that all
-`dockerTag` values look like `git-$COMPONENT_SHA_SHORT`, where `$COMPONENT_SHA_SHORT` is the first
-7 characters of the applicable SHA that you got in the previous step.
-4. Ensure that all `DEBUG` variables in manifests are `false`.
-5. Delete the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var from `tpl/deis-controller-rc.yaml`
```
Comment (helgi) [inline src/roadmap/release-checklist.md]: I assume these are all now accounted for in `deisrel` ?

Comment (bacongobbler) [inline src/roadmap/release-checklist.md]: yes. `deisrel` automates this process as seen below.

Comment (helgi) [inline src/roadmap/release-checklist.md]: Nevermind, i see it is a manual step further below

File: src/roadmap/release-checklist.md [hunk 2]
```diff
@@ -34,43 +34,59 @@ A release consists of the following artifacts:
...
+  ```
+  cp -r workflow-dev workflow-$WORKFLOW_RELEASE_SHORT
+  cp -r workflow-dev-e2e workflow-$WORKFLOW_RELEASE_SHORT-e2e
+  ```
+3. For the next few steps, we'll work with [deisrel](https://github.com/deis/deisrel):
+  * Clone the repo and build the tool:
+  ```
+  git clone https://github.com/deis/deisrel
+  cd deisrel
+  make build
```
Comment (arschles) [inline src/roadmap/release-checklist.md]: can we change these steps to download the binary from bintray?

File: src/roadmap/release-checklist.md [hunk 3]
```diff
@@ -93,8 +109,8 @@ Amazon S3                           |
 !!! note
     If bugs are found and fixes are made, do the following:
 
-      - Update the appropriate docker tag(s) in the `generate_params.toml` file
-      - Run `make git-update` in the aforementioned `deis-workflow-group` repository
+      - Update the appropriate docker tag(s) in the `generate_params.toml` ffile
```
Comment (arschles) [inline src/roadmap/release-checklist.md]: s/ffile/file

File: src/roadmap/release-checklist.md [hunk 4]
```diff
@@ -34,43 +34,59 @@ A release consists of the following artifacts:
...
-5. Delete the `KUBERNETES_POD_TERMINATION_GRACE_PERIOD_SECONDS` env var from `tpl/deis-controller-rc.yaml`
-6. Commit your changes: `git commit -a -m "chore(workflow-$WORKFLOW_RELEASE_SHORT): releasing workflow-$WORKFLOW_RELEASE_SHORT"`
-7. Push your changes to your fork: `git push -u $YOUR_FORK_REMOTE release-$DEIS_RELEASE`. Note that
-`$YOUR_FORK_REMOTE` is the git URI to the remote of your `deis/charts` fork. Mine is `git@github.com:arschles/deis-charts.git`, for example.
-8. Do steps 2-5 with the `workflow-beta3-e2e` directory
-9. Open a pull request from your branch to merge into `master` on https://github.com/deis/charts
+Next, we'll create new [Helm Classic](https://github.com/helm/helm-classic) charts so that we can "stage" a
+version of our release for testing. Here is the current process to do so:
+
+1. Create a new branch in [deis/charts](https://github.com/deis/charts): `git checkout -b release-$WORKFLOW_RELEASE`
```
Comment (arschles) [inline src/roadmap/release-checklist.md]: should this be `git checkout -b release-$WORKFLOW_RELEASE deis/master` for clarity?

Comment (vdice) [inline src/roadmap/release-checklist.md]: good call; added `origin/master` to represent `<start point>`

File: src/roadmap/release-checklist.md (hunk 2 above)
Comment (vdice) [inline src/roadmap/release-checklist.md]: ah yes, thank you for reminder; amended

**Pull request merged**